roles/ceph-osd/tasks/docker/start_docker_osd.yml

```diff
@@ -22,7 +22,6 @@
 - name: prepare ceph osd disk
   shell: |
     docker run --net=host \
-    --pid=host \
```

Comment:
I think it's fine and safe removing these 2 from prepare since this container doesn't run the OSD and is ephemeral.